changeset:   715:41fa04634db2
tag:         tip
user:        Jakub Wilk <jwilk@jwilk.net>
date:        Sat Oct 25 19:59:32 2014 +0200
summary:     Always pass ssl_options to LWP >= 6.

diff -r 78985f550841 -r 41fa04634db2 mbank-cli
--- a/mbank-cli	Sat Oct 25 19:53:40 2014 +0200
+++ b/mbank-cli	Sat Oct 25 19:59:32 2014 +0200
@@ -463,7 +463,8 @@
         IO::Socket::SSL::set_args_filter_hack('use_defaults');
         # TODO: Work around the bug also for earlier versions of
         # IO::Socket::SSL, which don't support set_args_filter_hack().
-    } elsif ($LWP::UserAgent::VERSION >= 6) {
+    }
+    if ($LWP::UserAgent::VERSION >= 6) {
         $ua->ssl_opts(
             @ssl_options,
         )

